appendix/basic_commands.tex

```diff
@@ -2445,8 +2448,8 @@ \subsection{RESTORE}
 
 The {\ttfamily RESTORE} statement resets where the {\ttfamily READ} statement
 will read from, causing the next {\ttfamily READ} to read from the first
-{\ttfamily DATA} statement again.  In this way, a program and re-read data that
-is has already read.  The {\ttfamily RESTORE} statement takes no arguments, and
+{\ttfamily DATA} statement again.  In this way, a program can re-read data that
+it has already read.  The {\ttfamily RESTORE} statement takes no arguments, and
```

Comment:
Why I need proof-readers!  Thanks!

Reply:
Actually, RESTORE can now take one argument: a line number. If RESTORE is used with a line number, that moves the READ pointer to the start of that line. 

Consider this trivial example:
```
10 PRINT 1
20 PRINT 2
30 RESTORE 20:READ A:PRINT A
```
This will print 2 when run.

Reply:
I think you mean `DATA` rather than `PRINT` :)

Reply:
>I think you mean DATA rather than PRINT :)

I do not know to what you refer. That was a perfect comment.